en/advanced-topics/dynamic-atlas.md

```diff
@@ -2,25 +2,30 @@
 
 > Authors: Xunyi, youyou
 
-## Introduction
-
 Reducing DrawCall is a very straightforward and effective way to improve game rendering efficiency, and a very important factor in whether the two DrawCall can be combined into a DrawCall is whether the two DrawCall use the same texture.
 
-Cocos Creator provides a static atlas packing when building a project -- **Auto Atlas**. But when the project grows bigger, the texture will become so much that it's hard to package the textures into a large texture. At this time, the static atlas packing is more difficult to meet the needs of reducing DrawCall. So Cocos Creator added the **Dynamic Atlas** feature to v2.0. It dynamically merges textures into a large texture while the project is running. When rendering to a texture, the Dynamic Atlas Manager will automatically detects if the texture has been added to the Dynamic Atlas Manager, if not, and the texture conforms to the Dynamic Atlas condition, then the texture will be merged into the large texture generated by the Dynamic Atlas Manager.
+Cocos Creator provides a static atlas packing when building a project -- **Auto Atlas**. But when the project grows bigger, the texture will become so much that it's hard to package the textures into a large texture. At this time, the static atlas packing is more difficult to meet the needs of reducing DrawCall. So Cocos Creator added the **Dynamic Atlas** feature to v2.0. It dynamically merges textures into a large texture while the project is running. When rendering to a texture, the Dynamic Atlas Manager will automatically detects if the texture has been merged into the Atlas (Collection of images). If not, and the texture conforms to the Dynamic Atlas condition, then the texture will be merged into the Atlas.
 
-**Dynamic Atlas** is according to the rendering order to determine whether the texture is merged into a large texture. This ensures that adjacent DrawCall can be combined into a single DrawCall.
+Dynamic Atlas is according to the rendering order to determine whether the texture is merged into a large texture. This ensures that adjacent DrawCall can be combined into a single DrawCall (Also known as Batching).
 
-## Use details
+## Enable, disable Dynamic Atlas
 
-Currently, the Dynamic Atlas Manager only supports the pack of the Sprite render component. If you want to add support for other types of render components, you can customize the rendering component yourself:
+During the initialization process of Cocos Creator, different [CLEANUP_IMAGE_CACHE](https://docs.cocos.com/creator/api/zh/classes/macro.html#cleanupimagecache) parameters will be set according to different platforms. When `CLEANUP_IMAGE_CACHE` is disabled, Dynamic Atlas will be enabled by default.
+
+**If you want to force enabled the Dynamic Atlas**, write it in your code:
```

Comment:
If you need to force the Dynamic Atlas to be enabled, execute the following code